core/pom.xml

```diff
@@ -150,6 +150,11 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.python</groupId>
+            <artifactId>jython</artifactId>
+            <version>2.7.0</version>
```

Comment:
I'll add it as a test-only dependency instead, to validate that we can call additional JSR-223 languages if required, but document that we only support JavaScript (as either Rhino in 7.x or Nashorn in 8.x) by default,